web-src/src/common/components/textfield.vue

```diff
@@ -191,8 +192,15 @@ export default {
   }
 }
 
-function getValidByTypeError(value, type, min, max, max_length) {
+function getValidByTypeError(value, type, min, max, max_length, regex) {
   if (type === 'text') {
+    if (regex) {
```

Comment:
Could you add a test to https://github.com/bugy/script-server/blob/master/web-src/tests/unit/textfield_test.js ?

Reply:
How do I get started with tests? How do I run them locally on my dev env?
It would be nice to document this on the Wiki somewhere :-)